CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+7.2.0
+-----
+* Add application_name configuration option
+* Update login page copy
+
```

Comment:
Ah, if you haven't released 7.1 then I'll sneak into that one. I didn't want to bump to `7.1.1` since it's not just a copy change.